### PR TITLE
Remove duplicate content and add links to readthedocs

### PR DESCRIPTION
**Pull request by eslesar-aws**
*Issue #, if available:*

Removed duplicated content from the main README.rst, /tensorflow/README.rst, and /sklearn/README.rst files and added links to the published content at ReadTheDocs.

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x ] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [x ] I have updated the [changelog](https://github.com/aws/sagemaker-python-sdk/blob/master/CHANGELOG.rst) with a description of my changes (if appropriate)
- [x ] I have updated any necessary [documentation](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: FAILED
* [Build Logs](https://9dzx6wxcy9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=4c9ed34c-dabe-42a0-8926-12f3e525a3dd%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: FAILED
* [Build Logs](https://9dzx6wxcy9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=a448fe13-7bc2-410c-9416-f5479a1146c9%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: FAILED
* [Build Logs](https://9dzx6wxcy9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=4aa76537-8335-4e01-b6a1-baf557322de4%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: FAILED
* [Build Logs](https://9dzx6wxcy9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=47709bfc-afa3-4a4c-a829-d796ba37346c%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: FAILED
* [Build Logs](https://9dzx6wxcy9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=e4a703e3-ce69-470e-80b3-f7ed84883320%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: SUCCEEDED
* [Build Logs](https://9dzx6wxcy9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=44b174a8-a8e6-4227-b463-4990e43cce69%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: SUCCEEDED
* [Build Logs](https://9dzx6wxcy9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=41a5bb02-02e8-44e5-8175-fb7e5eec2180%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: doc/overview.rst [hunk 1]
```diff
@@ -676,12 +676,3 @@ For comprehensive examples on how to use Inference Pipelines please refer to the
 - `inference_pipeline_sparkml_xgboost_abalone.ipynb <https://github.com/awslabs/amazon-sagemaker-examples/blob/master/advanced_functionality/inference_pipeline_sparkml_xgboost_abalone/inference_pipeline_sparkml_xgboost_abalone.ipynb>`__
 - `inference_pipeline_sparkml_blazingtext_dbpedia.ipynb <https://github.com/awslabs/amazon-sagemaker-examples/blob/master/advanced_functionality/inference_pipeline_sparkml_blazingtext_dbpedia/inference_pipeline_sparkml_blazingtext_dbpedia.ipynb>`__
 
-
```
Comment (nadiaya) [inline doc/overview.rst]: why are we removing this part?

Comment (eslesar-aws) [reply doc/overview.rst]: If you mean the above, it's just an extra blank line.

If you mean Workflow, it hasn't been migrated yet, so it needs to stay on the GitHub page, not in overview.rst

Comment (laurenyu) [reply doc/overview.rst]: I think it'd be better to keep the Workflow section but replace the link with an absolute link to the GH readme, and then update that link once the Workflow readme has been migrated

Comment (eslesar-aws) [reply doc/overview.rst]: Fair enough. I can migrate workflow next, so that it is there before the next release.

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: SUCCEEDED
* [Build Logs](https://9dzx6wxcy9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=78f2a4f2-b9f4-467e-b56f-b65131c562b1%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Pull request merged**